.changeset/light-eyes-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+"@capacitor-firebase/authentication": minor
```

Comment:
```suggestion
"@capacitor-firebase/authentication": patch
```

Please change this to `patch` as we are still in the development version (`0.x.y`).